assets/stylesheets/screens/keyword.scss

```diff
@@ -0,0 +1,12 @@
+.form-keyword-upload {
```

Comment:
These styles are not scoped to a specific screen. See my below comment on `layout/application.html` to resolve this.

Reply:
Fixed in this [comment](https://github.com/gutakk/go-google-scraper/pull/27#discussion_r539145928)